### ISSUE TITLE
is PyFlyff compatible with Macos?

### ISSUE THREAD

**Issue by direrFlood** [ils94/PyFlyff#27]
Hey,
is PyFlyff compatible with macOS?

**Comment by ils94** [comment]
> Hey, is PyFlyff compatible with macOS?

Nope because I use win32api to control alt windows

**Comment by direrFlood** [comment]
> > Hey, is PyFlyff compatible with macOS?
> 
> Nope because I use win32api to control alt windows

Is macOS fork possible?

**Comment by ils94** [comment]
> > > Hey, is PyFlyff compatible with macOS?
> > 
> > 
> > Nope because I use win32api to control alt windows
> 
> Is macOS fork possible?

yes, but to make use of the mini ftool and alt control features, you will have to find a similar library that gives the same capabilities as pywin32 but for MacOS, the same for Linux.
